### ISSUE TITLE
ele ainda permanece funcional? 

### ISSUE THREAD

**Issue by TaylorAvila** [lguilhermee/EAC-Extractor-Utility#3]

**Comment by lguilhermee** [comment]
Sim, porem a assinatura quebrou em algumas versoes do EAC EOS, mas segue a mesma encryptacao.

**Comment by TaylorAvila** [comment]
os padrões como //5. Encontre o padrão dos dados iniciais e o tamanho dos dados
    autoresultado =Digitalização de padrões((uintptr_t)lançadorDados,"A7 ED 96 0C 0F"); não deveriam mudar de acordo com cada game?

**Comment by TaylorAvila** [comment]
se tiver um discord para me dar um suporte lhe agradeço

**Comment by lguilhermee** [comment]
> os padrões como //5. Encontre o padrão dos dados iniciais e o tamanho dos dados
> 
>     autoresultado =Digitalização de padrões((uintptr_t)lançadorDados,"A7 ED 96 0C 0F"); não deveriam mudar de acordo com cada game? 

Não, esse é um padrão, vai encontrar em todos arquivos do eac
